client/package.json

```diff
@@ -5,9 +5,8 @@
   "sideEffects": false,
   "private": true,
   "scripts": {
-    "start-local-features": "yarn start-local -- -- --env $1",
```

Comment:
If we're removing this script, should also update readme

Reply:
Good catch, I wonder how/why that's being removed?

Reply:
removed because the script list was getting bloated, with the addition of a `start-remote-features` as well.
I had a look, and decided that `start-local` can be adjusted to suit both modes, no params passed and feature params passed; so there's no need to have both `start-local` and `start-local-features`

Reply:
Looks like it was a suggestion from Mark...

Reply:
I've updated the README, and will let the team know.